src/actuator/ActuatorFAST.C

```diff
@@ -214,6 +194,18 @@ ActuatorFAST::load(const YAML::Node& y_node)
             actuatorFASTInfo->fileToDumpPoints_ = turbFileName;
           }
 
+          // The correction from filtered lifting line theory
+          //~ const bool fllt_correction = 
+                  //~ cur_turbine["fllt_correction"].as<bool>();
+          bool fllt_correction=false;
+          get_if_present(cur_turbine, "fllt_correction", fllt_correction);
```

Comment:
You most likely want `get_if_present_no_default` here, or use 

```C++
get_if_present(cur_turbine, "fllt_correction", fllt_correction, fllt_correction);
```

Reply:
done